changelog/pop-grapheme.dd

```diff
@@ -0,0 +1,38 @@
+Added popGrapheme function to std.uni.
+
+The new function is a cross between the existing $(REF graphemeStride, std,
+uni) and $(REF decodeGrapheme, std, uni) functions. The new function both
+supports `@safe pure nothrow @nogc` like `graphemeStride` does as long as you
+don't rely on autodecoding (side node: `@nogc` support for `graphemeStride`
```

Comment:
This side note sounds like it should be its own PR.

Reply:
It [is](https://github.com/dlang/phobos/pull/9050). I felt an added `@nogc` compability to an existing funtion alone wasn't worth a changelog entry so didn't add one in that PR. But since I'm writing here that `graphemeStride` is `@nogc` anyway, which until very recently wasn't the case, I feel it's useful to mention it. As in: "this note applies only with the latest compiler/Phobos".